### ISSUE TITLE
publish unit test result action can't be used in macOS

### ISSUE THREAD

**Issue by jingchen2222** [4paradigm/HybridSE#70]
Issue tracker is **ONLY** used for reporting bugs. New features should be discussed on our discussion

# publish-unit-test-result-action can't be used in mac os

## Current Behavior

https://github.com/4paradigm/HybridSE/runs/2472541674?check_suite_focus=true
```
Run EnricoMi/publish-unit-test-result-action@v1
  //...
Error: Container action is only supported on Linux
```
## Possible Solution

Maybe find another action?
## Steps to Reproduce

1.
2.
3.
4.

**Comment by aceforeverd** [comment]
it's a docker action, not work on mac.

**Comment by aceforeverd** [comment]
I think it will fix in #69 
since it upload test result from compile env, and publish result from linux. Compile env do not matters!

**Comment by jingchen2222** [comment]
resolved by #67  #69

**Comment by jingchen2222** [comment]
resolved by #67 #69
